### PR TITLE
New post on finishing fast

### PR DESCRIPTION
**Pull request by steve-at-hx**
### Blog title
Finishing fast

### Blog description
Post on the role of speed in finishing fast, using running as an example.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [ ] I have checked for grammatical errors
- [ ] I have checked for spelling errors
- [ ] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2016-07-20-finishing-fast.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+author: steve_cs
+tags: Speed Pace Fast Finishing Projects 
+comments: true
+excerpt: A recent example, albeit nothing to do with business or technology, to remind us that speed is not the only element in finishing fast. 
+---
+We’ve been discussing speed a lot internally both in relation to our systems and our delivery. We want to waste no time in delivering new features and improvements to our customers to maintain our competitive advantage. I was grateful for a recent example, albeit nothing to do with business or technology, to remind us that speed is not the only element in finishing fast. 
+
+I’m not much of a runner, but I’ve been bringing my time down consistently over the last month. Imagine my delight when the sun was out, the air was clear and I was feeling fit. Conditions were perfect for a new personal best.
+
+##Speed, speed, speed
```
Comment (Joezo) [inline _posts/2016-07-20-finishing-fast.md]: You need a space after the hashes for jekyll to render the post properly :( 
https://cloudup.com/cP3rKfMXJcf

Comment (steve-at-hx) [inline _posts/2016-07-20-finishing-fast.md]: Done. Thanks, Joe.

**Comment by pmcnr-hx** [comment]
I would probably dot a few commas around, but that might possibly just be personal preference. It's great effort and reads very well: 👍

(you can also remove the blank lines at the end)

**Comment by Joezo** [comment]
Looks great 👍

**Pull request merged**